geomstats/_backend/numpy/autodiff.py

```diff
@@ -4,6 +4,8 @@
 The following functions return error messages.
 """
 
+from geomstats.exceptions import AutodiffNotImplementedError
```

Comment:
Why do we need a special exception, since the MSG is not the exception file?

Either:
1. Put the message in exception file
OR:
2. Only use RunTimeError with this MSG?
I'd prefer the second one, as it is easier for the reader.

If we keep that tailored exception, maybe change the name to:
UseAutodiffBackend?
Or
AutodiffNotInNumpy?